README.md

```diff
@@ -3,54 +3,134 @@
+ * [Using the application](#using-the-application)
+ * [Other use cases](#other-use-cases)
+ * [Developing the frontend code](#developing-the-frontend-code)
+* [End user documentation](#end-user-documentation)
 
-### Build the application
-_(All commands from here on will be assumed to be ran from the project's root directory)_
+## Contributing to Pantheon
+
+### Prerequisites
+
+* You must have an account on GitHub.  
+  [Signing up for a new GitHub account](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)
+* You must have registered SSH keys in your GitHub account.  
+[Adding a new SSH key to your GitHub account](https://help.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)
+* You must be a member of the `pantheon-doc-authors` team in the `redhataccess` group in GitHub.
+* You must be logged in to your account on GitHub.
+
+### Forking the repository
+
+Fork the repository so that you can create and work with branches independently of the `redhataccess/pantheon` repository.
+
+1. In a web browser, navigate to https://github.com/redhataccess/pantheon
+1. Click **Fork**.
+1. Click your user space in GitHub.
+
+### Cloning the repository
+
+After you have forked the repository, you must clone it to your local machine and add the original `redhataccess/pantheon` repository as an upstream remote.
+
+1. From a terminal, clone the repository:  
+       $ git clone git@github.com:<user-space>/pantheon.git
+1. Set up `redhataccess/pantheon` as the upstream:  
+       $ cd pantheon
+       $ git remote add upstream git@github.com:redhataccess/pantheon.git
+
+### Creating a working branch
+
+Whenever you work on a new issue, you must create a new working branch based on the latest version of the upstream master branch.
+
+1. Ensure you are on master  
+       $ git checkout master
+1. Ensure your fork is up to date  
+       $ git pull upstream master
+1. Create a working branch based on the issue in JIRA:  
+       $ git checkout -b FCCEUD<ID#>
+
+### Creating a pull request and completing review
+
+When your work is ready to be reviewed and merged, create a pull request.
+
+1. Push your working branch to your fork:  
+       $ git push -u origin <branch_name>
+1. From the repository page in GitHub, click **New pull request**.
+1. Select your working branch from the compare list.
+1. Add `WIP` to the title of the pull request.
+1. Add the **awaiting tech review** label to the pull request.
+1. In the pull request comment field, enter `@redhataccess/pantheon-developers Please review for technical completeness and accuracy`.
+1. Click **Create new pull request**.
+
+For code pull requests, one or more developers review the pull request. For documentation pull requests, the developers review the pull request for technical accuracy and documentation team members review the pull request for clarity and consistency and compliance with necessary standards.
```

Comment:
"clarity, consistency, and compliance with necessary standards"

Reply:
Fixed.